packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -216,6 +216,9 @@ ${lightGreen.wrap('🐦 Shorebird initialized successfully!')}
 ✅ A "shorebird.yaml" has been created.
 ✅ The "pubspec.yaml" has been updated to include "shorebird.yaml" as an asset.
 
+shorebird.yaml does not contain any sensitive information and should be checked into version control.
+You can freely share your app_id with others.
```

Comment:
I think this is good, but it woudl be better to just put these as comments in shorebird.yaml.

Some might also view "share your app_id" as sharing app_id between apps?

Reply:
It looks like we already say "this isn't secret" in the yaml comments. Updated those to be (I hope) clearer.